pilot/control/payload.py

```diff
@@ -202,7 +219,7 @@ def process_job_report(job):
     :return:
     """
 
-    log = get_logger(job.jobid)
+    log = logger.getChild(job.jobid)
```

Comment:
getChild() is not python 2.6 compatible which we must support since there are a few queues that won't update until next year. Please revert to use get_logger() which will use getChild() if available.

Reply:
ok. I will update it.
I will also update get_logger to add logger as a parameter. Otherwise all components will print pilot.util.auxiliary as its component, such as the example below where job_monitor's component should be pilot.control.job.

2018-07-22 18:21:24,114 | INFO     | job_monitor         | pilot.util.auxiliary.4000164444  | job_monitor               | monitor loop #5: job 0:4000164444 is in state 'finished'